packages/headless/src/app/engine.test.ts

```diff
@@ -259,4 +261,23 @@ describe('engine', () => {
       });
     });
   });
+
+  it('should exposes a Relay instance', () => {
+    const engine = initEngine();
+    expect(engine.relay).toBeDefined();
+  });
+
+  it('should return the same instance of Relay if the state has not changed', () => {
+    const engine = initEngine();
+    expect(engine.relay).toBe(engine.relay);
```

Comment:
That seems like a useless test 🤔  ?

Reply:
Not really: This test shows that if we call the relay getter twice for a given state, we get the same instance of Relay.

Reply:
Ok I understand. 
Then maybe just a small comment that explains this would help future readers